### PR TITLE
testing golang linter fix

### PR DESCRIPTION
**Pull request by wickedknock**
Updated super linter yaml to download go dependencies

### CODE REVIEW

**Comment by jbampton** [comment]
Can you fix the conflicts ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -204,10 +204,7 @@ jobs:
           keys:
             - v1-pkg-cache
       - run: go install github.com/mgechev/revive@latest
-      - run:
-          name: 🧹 Run Golang revive lint
-          command: |
-            sh ./check-revive.sh || exit 1
+
```
Comment (jbampton) [inline .circleci/config.yml:207]:
```suggestion
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -102,7 +102,7 @@ repos:
     hooks:
       - id: codespell
         entry: codespell -I codespell.txt ./doc/source
-        exclude: ^assets/.*$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^logorama/.*$|rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$
+        exclude: ^assets/.*$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^go.work.sum$|^logorama/.*$|^rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$
```
Comment (jbampton) [inline .pre-commit-config.yaml:105]:
```suggestion
        exclude: ^assets/.*$|^charts/.*$|^docs/assets/.*$|^eclipse-cheatsheets-to-dita-to-pdf/.*$|^Gemfile\.lock$|^go\.work\.sum$|^logorama/.*$|^rubycritic/.*$|^ruby-eclipse-cheatsheets-to-dita/.*$|^tests/spelling/.*$
```

File: macewindu/main.go [hunk 3]
```diff
@@ -59,25 +61,33 @@ func main() {
 	})
 
 	router.GET("/google-charts", func(c *gin.Context) {
-		r := rand.New(rand.NewSource(time.Now().UnixNano()))
 
-		offsetNumber := r.Intn(7)
+		max := big.NewInt(7)
+		offsetNumber, _ := rand.Int(rand.Reader, max)
+
```
Comment (jbampton) [inline macewindu/main.go:67]:
```suggestion
```

File: scrapers/go/colly/github/main.go [hunk 4]
```diff
@@ -14,37 +14,37 @@ import (
...
 	if err != nil {
 		log.Fatal(err)
 	}
-	record := scrape(username, delay)
-	fmt.Println(record)
+
 }
 
-func scrape(username string, delay int) []string {
+func Scrape(username string, delay int) []string {
```
Comment (jbampton) [inline scrapers/go/colly/github/main.go:47]:
```suggestion
// Main scraper function that gets all the stats
func Scrape(username string, delay int) []string {
```

File: scrapers/go/colly/github/main.go [hunk 5]
```diff
@@ -14,37 +14,38 @@ import (
...
 	if err != nil {
 		log.Fatal(err)
 	}
-	record := scrape(username, delay)
-	fmt.Println(record)
+
 }
 
-func scrape(username string, delay int) []string {
+// Main scraper function that gets all the stats
```
Comment (jbampton) [inline scrapers/go/colly/github/main.go:47]:
```suggestion
// Scrape is the main function that gets all the stats
```

File: .circleci/config.yml [hunk 6]
```diff
@@ -204,10 +204,6 @@ jobs:
           keys:
             - v1-pkg-cache
       - run: go install github.com/mgechev/revive@latest
-      - run:
-          name: 🧹 Run Golang revive lint
-          command: |
-            sh ./check-revive.sh || exit 1
```
Comment (jbampton) [inline .circleci/config.yml:210]: Why is this block removed ?

Comment (wickedknock) [reply .circleci/config.yml:210]: Ok I checked this was by mistake I selected this code also , I should not have , want me to add it back ?

Comment (wickedknock) [reply .circleci/config.yml]: added

**Pull request merged**